backend/models/UserSchema.js

```diff
@@ -1,6 +1,10 @@
 const mongoose = require("mongoose");
 
 const userSchema = new mongoose.Schema({
+  _id: {
```

Comment:
rename this to something else like user_id uid to not conflict with the mongo default generated field of _id